packages/react-components/react-overflow/stories/Overflow/CustomDividers.stories.tsx

```diff
@@ -0,0 +1,169 @@
+          <Button>Item 1</Button>
+        </OverflowItem>
+        <OverflowGroupDivider groupId={'1'} />
+        <OverflowItem id={'2'} groupId={'2'}>
+          <Button>Item 2</Button>
+        </OverflowItem>
+        <OverflowGroupDivider groupId={'2'} />
+        <OverflowItem id={'3'} groupId={'3'}>
+          <Button>Item 3</Button>
+        </OverflowItem>
+        <OverflowItem id={'4'} groupId={'3'}>
+          <Button>Item 4</Button>
+        </OverflowItem>
+        <OverflowGroupDivider groupId={'3'} />
+        <OverflowItem id={'5'} groupId={'4'}>
+          <Button>Item 5</Button>
+        </OverflowItem>
+        <OverflowItem id={'6'} groupId={'4'}>
+          <Button>Item 6</Button>
+        </OverflowItem>
+        <OverflowItem id={'7'} groupId={'4'}>
+          <Button>Item 7</Button>
+        </OverflowItem>
+        <OverflowGroupDivider groupId={'4'} />
+        <OverflowItem id={'8'} groupId={'5'}>
+          <Button>Item 8</Button>
+        </OverflowItem>
+        <OverflowMenu
+          itemIds={['1', 'divider-1', '2', 'divider-2', '3', '4', 'divider-3', '5', '6', '7', 'divider-4', '8']}
+        />
+      </div>
+    </Overflow>
+  );
+};
+
+const OverflowGroupDivider: React.FC<{
+  groupId: string;
+}> = props => {
+  return (
+    <OverflowDivider groupId={props.groupId}>
+      <div>
+        <ChevronRight20Regular />
+      </div>
+    </OverflowDivider>
+  );
+};
+
+const OverflowMenu: React.FC<{ itemIds: string[] }> = ({ itemIds }) => {
+  const { ref, overflowCount, isOverflowing } = useOverflowMenu<HTMLButtonElement>();
+
+  if (!isOverflowing) {
+    return null;
+  }
+
+  return (
+    <Menu>
+      <MenuTrigger disableButtonEnhancement>
+        <MenuButton ref={ref}>+{overflowCount} items</MenuButton>
+      </MenuTrigger>
+
+      <MenuPopover>
+        <MenuList>
+          {itemIds.map(i => {
+            // This is purely a simplified convention for documentation examples
+            // Could be done in other ways too
+            if (typeof i === 'string' && i.startsWith('divider')) {
+              const groupId = i.split('-')[1];
+              return <OverflowMenuDivider key={i} id={groupId} />;
+            }
+            return <OverflowMenuItem key={i} id={i} />;
+          })}
+        </MenuList>
+      </MenuPopover>
+    </Menu>
+  );
+};
+
+const OverflowMenuItem: React.FC<{ id: string }> = props => {
+  const { id } = props;
+  const isVisible = useIsOverflowItemVisible(id);
+
+  if (isVisible) {
+    return null;
+  }
+
+  return <MenuItem>Item {id}</MenuItem>;
+};
+
+const OverflowMenuDivider: React.FC<{
+  id: string;
+}> = props => {
+  const isGroupVisible = useIsOverflowGroupVisible(props.id);
+
+  if (isGroupVisible === 'visible') {
+    return null;
+  }
+
+  return <MenuDivider />;
+};
+
+CustomDividers.parameters = {
+  docs: {
+    description: {
+      story: ['OverflowDivider should be used for larger dividers to include its width to the calculation.'].join('\n'),
```

Comment:
Let's be more descriptive about this, you should:
* describe the problem
* describe why simple padding is enough like the current divider story

Reply:
fixed